### PR TITLE
[e2e-test] Enhance loginAsGithubUser() to Handle Intermittent Pop-up Visibility Issues & Fix Quay test

### PR DESCRIPTION
**Pull request by subhashkhileri**
## Description

[e2e-test] Enhance loginAsGithubUser() to Handle Intermittent Pop-up Visibility Issues

## Which issue(s) does this PR fix

https://issues.redhat.com/browse/RHIDP-1396

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by invincibleJai — APPROVED** [review]
/lgtm

File: .ibm/pipelines/openshift-tests.sh [hunk 1]
```diff
@@ -8,8 +8,8 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 cleanup() {
   echo "Cleaning up before exiting"
-  helm uninstall ${RELEASE_NAME} -n ${NAME_SPACE}
```
Comment (gustavolira) [inline .ibm/pipelines/openshift-tests.sh:11]: @subhashkhileri why did you remove it?

Comment (subhashkhileri) [reply .ibm/pipelines/openshift-tests.sh:11]: The plan was to keep the instance running until another PR check is triggered, allowing us to inspect the cluster in case of failure. I've noticed that we've commented out `oc delete namespace ${NAME_SPACE}`, but not `helm uninstall ${RELEASE_NAME} -n ${NAME_SPACE}`. This approach keeps the namespace but uninstalls the RHDH.

**Comment by nickboldt** [comment]
Reminder that if you want changes like this in the 1.1.x branch, you have to cherrypick this change to that branch.

**Comment by subhashkhileri** [comment]
@nickboldt I have raised PR for 1.1.x branch : https://github.com/janus-idp/backstage-showcase/pull/996
